static/schemas/2.0/skus.2.0.json

```diff
@@ -0,0 +1,248 @@
+{
+    "title": "Skus.2.0",
+    "definitions": {
+        "SkusRules": {
+            "description": "Possible Rules a Sku can use",
+            "type": "array",
+            "items": {
+                "type": "object",
```

Comment:
@BillyAbildgaard : You can see the full supported keyword for each type from this link: http://json-schema.org/latest/json-schema-validation.html